### PR TITLE
Add decoder for liquity

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: rotkehlchen/chain/ethereum/decoding/liquity/decoder.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                event.event_type = HistoryEventType.SPEND
+                event.event_subtype = HistoryEventSubType.PAYBACK_DEBT
+                event.counterparty = CPT_LIQUITY
+                event.notes = f'Return {event.balance.amount} {A_LUSD.symbol} to liquity'
+            elif event.event_type == HistoryEventType.SPEND and event.event_subtype == HistoryEventSubType.NONE and event.asset == A_ETH:  # noqa: E501
+                event.event_type = HistoryEventType.DEPOSIT
+                event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                event.counterparty = CPT_LIQUITY
+                event.notes = f'Deposit {event.balance.amount} {event.asset.symbol} as collateral for liquity'  # noqa: E501
+            elif event.event_type == HistoryEventType.RECEIVE and event.event_subtype == HistoryEventSubType.NONE and event.asset == A_ETH:  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/decoding/liquity/decoder.py:54]: Is ETH the only collateral type in Liquity?

File: rotkehlchen/chain/ethereum/decoding/liquity/decoder.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                event.event_type = HistoryEventType.WITHDRAWAL
+                event.event_subtype = HistoryEventSubType.REMOVE_ASSET
+                event.counterparty = CPT_LIQUITY
+                event.notes = f'Withdraw {event.balance.amount} {event.asset.symbol} from collateral at liquity'  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/decoding/liquity/decoder.py:58]:
```suggestion
                event.notes = f'Withdraw {event.balance.amount} {event.asset.symbol} collateral from liquity'  # noqa: E501
```

**Review by LefterisJP — COMMENTED** [review]
minor comments

File: rotkehlchen/chain/ethereum/decoding/liquity/decoder.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/decoding/liquity/decoder.py:54]: yes

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/decoding/liquity/decoder.py:54]: Ah that's cool and easy then.

**Review by LefterisJP — APPROVED** [review]
LGTM fren.

**Pull request merged**